engine/schema/src/main/resources/META-INF/db/schema-41720to41800.sql

```diff
@@ -1561,7 +1561,6 @@ CREATE VIEW `cloud`.`user_view` AS
 DELETE FROM `cloud`.`snapshot_store_ref`
 WHERE store_role = "Primary" AND store_id IN (SELECT id FROM storage_pool WHERE removed IS NOT NULL);
 
-
```

Comment:
maybe undo this delete?

Reply:
@DaanHoogland I think the file indentation is better by removing this line.